bin/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "bin"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+libp2p = "0.41.0"
```

Comment:
Do we need these? If Cargo.toml of our binary is not ready, let's make it working before merging this PR.